public/stylesheet.css

```diff
@@ -264,16 +276,24 @@ a {
     margin: 0 0 1rem;
   }
 }
-
 .header__list-item {
-  font-family: var(--font-family-stylized);
-  font-size: 1rem;
-  font-weight: 600;
-  margin: 0 0.7rem;
+  height: 40px;
+  width: 40px;
+  margin: 10px;
+  padding: 5px;
+  fill: var(--color-menu-icon);
 }
-.header__list-item:hover {
-  color: var(--color-button-hover);
-  text-decoration: underline;
+
+.header__list-item:hover,
+.header__list-item:focus {
+  cursor: pointer;
+  height: 48px;
+  width: 48px;
+  margin: 6px;
+}
+
+.header__list-icon {
+  fill: var(--color-menu-icon) !important;
 }
```

Comment:
Am I mistaken or is this rule redundant given [line 284](https://github.com/simple-icons/simple-icons-website/blob/fdf6e6e4a60522671c27832831395cb142de9d70/public/stylesheet.css#L284) :thinking:

Reply:
that depends on whether we are going to go back to inverted colors.